### PR TITLE
feat:Remove bintray repository from gradle

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

**Review by HarryH96 — APPROVED** [review]

**Review by devmarcy — DISMISSED** [review]

**Review by sabahirfan — DISMISSED** [review]

File: charts/civil-service/values.preview.template.yaml [hunk 1]
```diff
@@ -128,6 +128,7 @@ ccd:
         DATA_STORE_IDAM_KEY: ${CCD_DATA_STORE_S2S_SECRET}
         DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,bulk_scan_orchestrator,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,civil_service
         IDAM_USER_URL: https://idam-web-public.aat.platform.hmcts.net
+        IDAM_OIDC_URL:  https://idam-web-public.aat.platform.hmcts.net
```
Comment (swalker125) [inline charts/civil-service/values.preview.template.yaml:131]: what's this change for @dharmendrak? Seem unrelated to bintray

Comment (dharmendrak) [reply charts/civil-service/values.preview.template.yaml:131]: ccd have changed this config
